ui/serializers.py

```diff
@@ -62,26 +62,40 @@ class Meta:
 class VideoSerializer(serializers.ModelSerializer):
     """Video Serializer"""
     key = serializers.SerializerMethodField()
+    collection_key = serializers.SerializerMethodField()
+    collection_title = serializers.SerializerMethodField()
     videofile_set = VideoFileSerializer(many=True, read_only=True)
     videothumbnail_set = VideoThumbnailSerializer(many=True, read_only=True)
 
     def get_key(self, obj):
         """Custom getter for the key"""
         return obj.hexkey
 
+    def get_collection_key(self, obj):
+        """Get collection key"""
+        return obj.collection.hexkey
+
+    def get_collection_title(self, obj):
+        """Get collection title"""
+        return obj.collection.title
+
     class Meta:
         model = models.Video
         fields = (
             'key',
             'created_at',
             'title',
             'description',
+            'collection_key',
+            'collection_title',
+            'multiangle',
```

Comment:
@aliceriot I think if you add `status` here that should make it available to the front end.